### PR TITLE
Adds a new contraband crate to cargo

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: 
add: Added a new contraband crate to cargo, the ATV crate.
/:cl:

the new crate requires an emagged/multitooled cargo console and costs 3000 credits.
it contains an ATV and a key.

### CODE REVIEW

**Comment by TehZombehz** [comment]
Only 2000?

**Comment by Tacolizard** [comment]
@TehZombehz most contraband crates are pretty expensive, and considering the fact that only one person can use the atv, i settled on 2000. i am open to suggestions.

**Comment by ExcessiveUseOfCobblestone** [comment]
does the atv even do anything?

**Comment by PKPenguin321** [comment]
It is basically a faster scooter that needs a key I think

**Comment by ExcessiveUseOfCobblestone** [comment]
didn't it have a use for a certain tile on the snow mission?

**Comment by Tacolizard** [comment]
so uh merge or no merge @Cheridan @optimumtact

**Comment by Shadowlight213** [comment]
@Tacolizard the minimum amount of time a pr needs to be open before a merge is 24 hours. It's only been 19.

**Comment by Tacolizard** [comment]
ok i literally didnt even know that was a rule, thanks @Shadowlight213

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm
Comment (Tacolizard) [inline code/modules/cargo/packs.dm]: i really wanna keep it cheap because it only has one ATV and greytiders everywhere will foam at the mouth when they see it.

**Comment by Tacolizard** [comment]
ok it's done. 3000 credits is the new cost.

**Comment by MimicFaux** [comment]
Cargonia bikergangs when?

**Comment by Tacolizard** [comment]
@optimumtact @Shadowlight213 @Cheridan 

**merge my meme ples?**

**Comment by optimumtact** [comment]
stop mergebegging, it only makes us hold your pr until even more time has elapsed.

We'll get to it when we can

**Comment by Tacolizard** [comment]
@optimumtact thats mean

**Comment by AdamElTablawy** [comment]
i cant wait to drive around on my atv as a gangster pulling drivebys on the hos with my uzi

but also you might wanna consider how relatively cheap this is, it'd be a staple order for any antag cargo

**Comment by optimumtact** [comment]
Every time you ping me I'm adding 24 hours to before I even bother looking at this.

This is a volunteer project, people will get to your pr when they can, hounding them about it will not help

**Comment by Shadowlight213** [comment]
Closed for mergebegging meme

**Comment by Iamgoofball** [comment]
Just wait for the first merge conflict THEN fix and merge beg

**Comment by Cheridan** [comment]
of all the dumb crap cargo can buy, THIS is super-expensive and contraband? 
can't you make a scooter that does the same thing for free?

**Comment by Iamgoofball** [comment]
This is faster

**Comment by Iamgoofball** [comment]
And it negates slowdown from damage

**Comment by Tacolizard** [comment]
@Cheridan the price was at 2000, i could put it back if this is too expensive. it's only contraband because it would be weird for stations to be able to buy atvs for no reason.

**Comment by Cheridan** [comment]
I guess that makes sense

**Pull request merged**
